test/quicklink.spec.js

```diff
@@ -139,11 +139,11 @@ mainSuite('should only prefetch links of same origin (default)', async () => {
   await page.goto(`${server}/test-same-origin.html`);
 
   await sleep(1000);
-  assert.equal(Array.isArray(responseURLs), true);
+  assert.is(Array.isArray(responseURLs), true);
   // => origins: [location.hostname] (default)
-  assert.equal(responseURLs.includes(`${server}/2.html`), true);
-  assert.equal(responseURLs.includes('https://example.com/1.html'), false);
-  assert.equal(responseURLs.includes('https://github.githubassets.com/images/spinners/octocat-spinner-32.gif'), false);
+  assert.is(responseURLs.includes(`${server}/2.html`), true);
+  assert.is(responseURLs.includes('https://example.com/1.html'), false);
+  assert.is(responseURLs.includes('https://github.githubassets.com/images/spinners/octocat-spinner-32.gif'), false);
```

Comment:
low prio alternatives

```suggestion
  assert.not.ok(responseURLs.includes('https://example.com/1.html'));
  assert.not.ok(responseURLs.includes('https://github.githubassets.com/images/spinners/octocat-spinner-32.gif'));
```